File: test/fixtures/output/java/restclient/full.java
RestClient restClient = RestClient.create();

MultiValueMap<String, String> formDataMap = new LinkedMultiValueMap<>();
formDataMap.add("foo", "bar");

ResponseEntity<String> response = restClient
  .method(HttpMethod.POST)
  .uri("http://mockbin.com/har", uriBuilder -> {
    uriBuilder.queryParam("foo", "bar");
    uriBuilder.queryParam("foo", "baz");
    uriBuilder.queryParam("baz", "abc");
    uriBuilder.queryParam("key", "value");
    return uriBuilder.build();
  })
  .cookie("foo", "bar")
  .cookie("bar", "baz")
  .header("accept", "application/json")
  .contentType(MediaType.APPLICATION_FORM_URLENCODED)
  .body(formDataMap)
  .retrieve()
  .toEntity(String.class);

File: test/fixtures/output/csharp/restsharp/full.cs
var client = new RestClient("http://mockbin.com");
var request = new RestRequest("/har?foo=bar&foo=baz&baz=abc&key=value", Method.Post);
request.AddHeader("accept", "application/json");
request.AddHeader("content-type", "application/x-www-form-urlencoded");
request.AddCookie("foo", "bar");
request.AddCookie("bar", "baz");
request.AddParameter("application/x-www-form-urlencoded", "foo=bar", ParameterType.RequestBody);
var response = client.Execute(request);